example/dune

```diff
@@ -1,6 +1,5 @@
 (executable
  (name mandels)
- (public_name ocaml-parmap-mandels)
```

Comment:
Without a `public_name`, the executable is never installed, and thus is not built by the default instructions in the opam file.

Reply:
Thanks a lot for helping out with this dune configuration :-)